src/lib/es2021.intl.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace Intl {

```

Comment:
This is missing `formatRange` & `formatRangeToParts` from `Intl.DateTimeFormat`
Ref: https://tc39.es/ecma402/2021/#sec-intl.datetimeformat.prototype.formatRange